pages/favorite.js

```diff
@@ -0,0 +1,43 @@
+import useLocalStorageState from "use-local-storage-state";
+import Navigation from "@/components/Navigation/Navigation";
+import FavoriteButton from "@/components/FavoriteButton/FavoriteButton";
+import {
+  StyledH1,
+  StyledImage,
+  ButtonContainer,
+  StyledLink,
+  StyledContainer,
+  ContentContainer,
+  MainContainer,
+} from "../components/Article.js/styles";
+
+export default function FavoritePage() {
+  const [favorites, setFavorites] = useLocalStorageState("favorites", []);
+
+  return (
+    <div>
+      <Navigation />
+      <div>
+        {favorites?.map(({ title, teaserImage, shareURL }) => (
+          <StyledContainer key={shareURL}>
```

Comment:
If you create a ``Article`` component, as I mentioned before, you could also use it here!